configs/skeleton/posec3d/stgcn_ntu-rgbd60_xsub.py

```diff
@@ -0,0 +1,74 @@
+model = dict(
+    type='ST_GCN_18',
+    in_channels=3,
+    num_class=60,
+    dropout=0.5,
+    edge_importance_weighting=True,
+    # graph_cfg=dict(layout='ntu-rgb+d', strategy='spatial')
+    graph_cfg=dict(layout='coco', strategy='spatial'))
+
+dataset_type = 'PoseDataset'
+ann_file_train = 'data/posec3d/ntu60_xsub_train.pkl'
+ann_file_val = 'data/posec3d/ntu60_xsub_val.pkl'
+train_pipeline = [
+    dict(type='UniformSampleFrames', clip_len=300),
```

Comment:
Better to use `padding_with_loop` to pad the sequence to length 300, which is the common practice for GCN.